apps/Learning-Buddy/navigation/AppStack.jsx

```diff
@@ -3,20 +3,22 @@ import { createStackNavigator } from '@react-navigation/stack';
 import { colors } from '../config/colors';
 import { QuestionForm } from '../screens/QuestionForm';
 import { HomeScreen } from '../screens/HomeScreen';
+import { NewQuizScreen } from '../screens/NewQuizScreen';
 
 const Stack = createStackNavigator();
 
 export const AppStack = () => {
   return (
     <Stack.Navigator
-      initialRouteName='Home Screen'
+      initialRouteName='New Quiz Screen'
```

Comment:
This needs to be changed back to home screen; if you need to adjust the buttons on that page to allow users to access this page, feel free, but the initial screen should be home.

![image](https://github.com/new-idea-machine/os-lesson-delivery-system/assets/105380332/59d3b8ad-b863-49c1-b0ce-22f2d52ea5b0)

Reply:
This is done

Reply:
Unresolving as a reminder to do this once you're ready to have merged again.

Reply:
Done